### ISSUE TITLE
Automatic info off does not automatically close the info dialog

### ISSUE THREAD

**Issue by rreinecke** [HydroSysPotsdam/Forester#44]
**Describe the bug**
Disabling the automatic information window does not lead to an automatic minimization. Only clicking on the i-icon minimizes the window.

**To Reproduce**
1. Open a project
2. Click on Automatic in the info window
3. Move the mouse somewhere else

**Expected behavior**
I expected the information window to hide automatically after disabling the automation. (Btw. it could be called "disable/enable hints" for clarity)

**Please add the following information:**
 - OS: Linux
 - Browser Firefox, Chrome
 - Python Version 3

**Comment by rreinecke** [comment]
Addition:

- Similar behavior with the options menu after clicking on a gear-icon: it is unclear what one needs to do to close it. Pressing escape works, but there should be a button or clicking somewhere else should close it automatically.

**Comment by david-strahl** [comment]
I agree, the functionality is unnecessarily complicated. The hint panel can always be opened and is then updated regardless of the automatic opening functionality.

**Proposed Solution**
- [ ] Will rename *automatic* field to *Show Hints*
- [ ] Hint panel will close when deactivated
